### PR TITLE
Bugfix/event order

### PR DESCRIPTION
**Pull request by pnuckowski**

### CODE REVIEW

File: alamo_scheduler/scheduler.py [hunk 1]
```diff
@@ -115,21 +115,39 @@ def schedule_job(self, check):
...
+                continue
+
+            if job is not None:
+                self.remove_job(check['id'])
+
             if any([trigger['enabled'] for trigger in check['triggers']]):
                 self.schedule_job(check)
 
-        logger.debug('Messages consumed.')
+        logger.debug('Consumed {} messages from kafka.'.format(len(checks)))
```
Comment (deejay1) [inline alamo_scheduler/scheduler.py]: so did you consume checks or messages? ;)

File: alamo_scheduler/scheduler.py [hunk 2]
```diff
@@ -115,21 +115,39 @@ def schedule_job(self, check):
...
+
+            if job is not None:
+                scheduled_check, = job.args
+
+            timestamp = scheduled_check.get('timestamp', 0)
+            # outdated message
+            if timestamp > check['timestamp']:
+                continue
+
+            if job is not None:
```
Comment (deejay1) [inline alamo_scheduler/scheduler.py]: we check that twice - do we need to?

**Comment by h00dy** [comment]
:+1:

File: alamo_scheduler/scheduler.py [hunk 3]
```diff
@@ -10,8 +10,9 @@
 from aiomeasures import StatsD
 from apscheduler.jobstores.base import JobLookupError
 from apscheduler.schedulers.asyncio import AsyncIOScheduler
-from kafka.client import KafkaClient
-from kafka.consumer.simple import SimpleConsumer
+
```
Comment (deejay1) [inline alamo_scheduler/scheduler.py]: needless space?

File: alamo_scheduler/scheduler.py [hunk 4]
```diff
@@ -10,8 +10,9 @@
...
+from kafka.consumer.kafka import KafkaConsumer
+
```
Comment (deejay1) [inline alamo_scheduler/scheduler.py]: and here ?

**Comment by deejay1** [comment]
:+1:

**Comment by h00dy** [comment]
:+1:

**Pull request merged**